### ISSUE TITLE
Exceptions thrown changing channel permissions

### ISSUE THREAD

**Issue by ZeroSumGame** [b1naryth1ef/dscord#21]
```
Task terminated with unhandled exception:
2017-02-05T18:34:45.746:client.d:run:360 failed to handle etc.linux.memoryerror.NullPointerError@src/etc/linux/memoryerror.d(325)
----------------
??:? void etc.linux.memoryerror.sigsegvUserspaceProcess(void*) [0xb41535]
??:? void etc.linux.memoryerror.sigsegvDataHandler() [0xb41476]
../src/dscord/state.d:204 void dscord.state.State.onChannelUpdate(dscord.gateway.events.ChannelUpdate) [0x90c7eb]
../src/dscord/util/emitter.d:126 _D6dscord4util7emitter7Emitter50__T6listenTC6dscord7gateway6events13ChannelUpdateZ6listenMFDFC6dscord7gateway6events13ChannelUpdateZvE6dscord4util7emitter12EmitterOrderZ57__T9__lambda3TS3std7variant18__T8VariantNVmi32Z8VariantNZ9__lambda3MFS3std7variant18__T8VariantNVmi32Z8VariantNZv [0x928d30]
../src/dscord/util/emitter.d:77 void dscord.util.emitter.EventListener.call(immutable(char)[], std.variant.VariantN!(32uL).VariantN) [0x91ce5d]
../src/dscord/util/emitter.d:172 _D6dscord4util7emitter7Emitter55__T10emitByNameTC6dscord7gateway6events13ChannelUpdateZ10emitByNameMFAyaC6dscord7gateway6events13ChannelUpdatebZ9__lambda4MFZv [0x91ed7a]
../../../.dub/packages/vibe-d-0.7.30/vibe-d/source/vibe/core/core.d:592 void vibe.core.core.makeTaskFuncInfo!(void delegate()).makeTaskFuncInfo(ref void delegate()).callDelegate(vibe.core.core.TaskFuncInfo*) [0x942bc4]
../../../.dub/packages/vibe-d-0.7.30/vibe-d/source/vibe/core/core.d:1221 void vibe.core.core.CoreTask.run() [0xa88332]
??:? void core.thread.Fiber.run() [0xb8f2af]
??:? fiber_entryPoint [0xb8f012]
??:? [0xffffffff] ({"t":"CHANNEL_UPDATE","s":4,"op":0,"d":{"type":0,"topic":"","position":4,"permission_overwrites":[{"type":"role","id":"256415537310072832","deny":0,"allow":805828689},{"type":"member","id":"mymemid","deny":0,"allow":805829713},{"type":"role","id":"269660067194732547","deny":0,"allow":0},{"type":"member","id":"memid","deny":0,"allow":805829713},{"type":"member","id":"memid","deny":0,"allow":805829713}],"name":"testingbot","last_message_id":"277869415192854529","id":"myid","guild_id":"myguildid"}})
```

**Comment by b1naryth1ef** [comment]
Should be fixed in 917875f

**Comment by ZeroSumGame** [comment]
Yep, that fixed it.   Thank you.  ;)
